apps/pipelines/nodes/nodes.py

```diff
@@ -261,6 +261,13 @@ class LLMResponseWithPrompt(LLMResponse, HistoryMixin):
             widget=Widgets.select, options_source=OptionsSource.collection, flag_required="document_management"
         ),
     )
+    document_index_id: OptionalInt = Field(
+        None,
+        title="Document Index",
```

Comment:
Shouldn't this be called 'Collection' to keep it consistent?

Reply:
https://github.com/dimagi/open-chat-studio/pull/1560/commits/fa1f31092fee54328bd2937cdcda911f300ba285